.github/workflows/CI.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'LICENSE.md'
+      - 'README.md'
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'LICENSE.md'
+      - 'README.md'
+  release:
+    types:
+      - created
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.3]
+        julia-arch: [x64, x86]
```

Comment:
x86 might cause problems in the tests - IIRC it did in the past and I just restricted the tests to run on x64.  It may be okay here but we should watch for the possibility of problems with it.